.storybook/Button/Button.stories.jsx

```diff
@@ -2,7 +2,8 @@ import React from "react";
 import Button from "./Button";
 
 export default {
-    title: "@Example|Button",
+    title: "Example/Button",
```

Comment:
[Hierarchy separators](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#removed-hierarchy-separators) changed back in Storybook 6.0. This is the correct way of using them now